test/DynamoCoreTests/DSLibraryTest.cs

```diff
@@ -290,5 +292,26 @@ public void TestBuiltinPackagesTokenIsFirstInList()
             // Check that builtinPackages token is first
             Assert.AreEqual(DynamoModel.BuiltInPackagesToken, customPackageFolders[0]);
         }
+
+        [Test]
+        [Category("UnitTests")]
+        public void Dispose_DS_ClassInstance()
+        {
+            CurrentDynamoModel.EngineController.LibraryServices.ImportLibrary("..\\..\\..\\test\\core\\library\\testclass.ds");
+
+            var testNode = new DSFunction(CurrentDynamoModel.LibraryServices.GetFunctionDescriptor("Test.CreateTest"));
+            CurrentDynamoModel.ExecuteCommand(new DynamoModel.CreateNodeCommand(testNode, 0, 0, true, false));
+            var node = CurrentDynamoModel.CurrentWorkspace.Nodes.FirstOrDefault();
+            Assert.IsNotNull(node);
+
+            int stackCount = CurrentDynamoModel.EngineController.LiveRunnerRuntimeCore.RuntimeMemory.Stack.Count;
+            CurrentDynamoModel.DeleteModelInternal(new List<Graph.ModelBase>() { node });
```

Comment:
Do these commands also execute the graph?

Reply:
Yes...both execute the graph